vars/sconsBuild.groovy

```diff
@@ -1,7 +1,5 @@
 // vars/sconsBuild.groovy
 
-import com.intel.checkoutScm
-
```

Comment:
Removing this leaves src/com/intel/checkoutScm.groovy and vars/checkoutScmWithSubmodules.groovy as dead code yes?  Shouldn't this patch remove them also then?